server/src/main/java/org/elasticsearch/indices/analysis/AnalysisModule.java

```diff
@@ -68,13 +70,18 @@ public final class AnalysisModule {
     private final HunspellService hunspellService;
     private final AnalysisRegistry analysisRegistry;
 
-    public AnalysisModule(Environment environment, List<AnalysisPlugin> plugins) throws IOException {
-        NamedRegistry<AnalysisProvider<CharFilterFactory>> charFilters = setupCharFilters(plugins);
+    public AnalysisModule(Environment environment, List<AnalysisPlugin> plugins, StablePluginsRegistry stablePluginRegistry)
+        throws IOException {
```

Comment:
Does it make sense to retain the 2-arg constructor, which can delegate to the 3-arg with a `new StablePluginsRegistry()`, as the third arg?

Reply:
na! just a q.